macros/stitch/base/stitch_fb_ad_creatives.sql

```diff
@@ -8,39 +8,55 @@
 {% macro default__stitch_fb_ad_creatives() %}
 
 with base as (
+    
+    select * from {{ var('ad_creatives_table') }}
+    
+),
+
+child_links as (
+    
+    select * from {{ ref('fb_ad_creatives__child_links') }}
+    
+),
 
+links_joined as (
+    
     select
     
         id as creative_id,
```

Comment:
you've added a join to this CTE, which now makes it very challenging to tell which fields come from which of the tables that are being joined. please prefix all fields with the table that they're coming from in this situation so that others can understand your code when they read through it!